jenkins-schema.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./jenkins.sh
```

Comment:
I think this needs a line like: https://github.com/alphagov/finder-frontend/blob/master/jenkins-schema.sh#L10

Otherwise how will the govuk-content-schemas commit on GitHub get the build status?